src/data/rat/defs.lean

```diff
@@ -585,3 +586,6 @@ lemma coe_int_inj (m n : ℤ) : (m : ℚ) = n ↔ m = n := ⟨congr_arg num, con
 end casts
 
 end rat
+
+-- Guard against import creep.
+assert_not_exists field
```

Comment:
`field` imports `rat`, right? So this sounds a bit redundant.

Reply:
It's the opposite assertion: that `field` hasn't been defined at this point.